pkg/dcgmexporter/dcgm.go

```diff
@@ -29,7 +36,14 @@ func NewGroup() (dcgm.GroupHandle, func(), error) {
 		return dcgm.GroupHandle{}, func() {}, err
 	}
 
-	return group, func() { dcgm.DestroyGroup(group) }, nil
+	return group, func() {
+		err := dcgm.DestroyGroup(group)
+		if err != nil {
```

Comment:
Just a nit:
if err := dcgm.DestroyGroup(group); err != nil`

Thanks for handling the return error.

Reply:
This is the point of the PR :) Linter detects such issues :)